main.go

```
@@ -53,20 +56,24 @@ func loadConfig() {
 }
 
 func setupLogger(cfg *config.Config) {
 	var err error
 	err = logger.Init(cfg.Log.LogFilePath, cfg.Log.MaxLogSize)
 	if err != nil {
-		log.Fatalf("Failed to initialize logger: %v", err)
+		fmt.Printf("Failed to initialize logger: %v\n", err)
 	}
-	logInfo("Config File Path: ", cfgfile)
-	logInfo("Loaded config: %v\n", cfg)
+	logger.SetLogLevel(cfg.Log.Level)
```

golangci-lint (1.62.2)

Error return value of `logger.SetLogLevel` is not checked

(errcheck)